### PR TITLE
v0.8.0 Non-Signed In User

### PR DESCRIPTION
**Pull request by henry-richard**
@wanda-carlson wants a description but its a self-explanatory pr, don't you think? 
Please tell me you agree! 

Screen Shot 2022-01-06 at 2 57 26 PM

### CODE REVIEW

**Comment by wanda-carlson** [comment]
@undefined can you add a description?

File: src/henry.py [hunk 1]
```diff
@@ -0,0 +1 @@
+print("henry testing v0.8.0")
```
Comment (wanda-carlson) [inline src/henry.py:1]: Henry is a name and should have a capital H. @henry-richard

**Review by wanda-carlson — CHANGES_REQUESTED** [review]

File: src/henry.py (hunk 1 above)
Comment (henry-richard) [reply src/henry.py:1]: So nitpicky!

**Review by wanda-carlson — APPROVED** [review]
All good

**Pull request merged**